domains/kennymolina.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+   "username": "xXMolinaXx",
+    "email": "kjmolina@live.com"
+  },
+  "record": {
+    "CNAME": "https://xxmolinaxx.github.io/"
```

`CNAME` record must contain a domain only, remove `https://` and `/`
```suggestion
    "CNAME": "xxmolinaxx.github.io"
```